src/splitInteger.test.js

```diff
@@ -2,20 +2,44 @@
 
 const splitInteger = require('./splitInteger');
 
-test(`should split a number into equal parts
-  if a value is divisible by a numberOfParts`, () => {
+test('should return equal parts', () => {
+  const result = splitInteger(50, 2);
 
+  expect(result).toEqual([25, 25]);
 });
 
-test(`should return a part equals to a value
-  when splitting into 1 part`, () => {
+test(`should return equal parts`, () => {
+  const result = splitInteger(8, 4);
 
+  expect(result).toEqual([2, 2, 2, 2]);
 });
 
-test('should sort parts ascending if they are not equal', () => {
+test(`should return parts`, () => {
+  const result = splitInteger(6, 1);
 
+  expect(result).toEqual([6]);
 });
 
-test('should add zeros if value < numberOfParts', () => {
+test('should return parts', () => {
+  const result = splitInteger(10, 3);
 
+  expect(result).toEqual([3, 3, 4]);
+});
+
+test('should sort parts', () => {
+  const result = splitInteger(17, 4);
+
+  expect(result).toEqual([4, 4, 4, 5]);
+});
+
+test('should add zeros', () => {
```

Comment:
'should add zeros if value < numberOfParts'

Reply:
Done